Index: /trunk/gdal/port/cpl_vsil_unix_stdio_64.cpp
===================================================================
--- /trunk/gdal/port/cpl_vsil_unix_stdio_64.cpp	(revision 38819)
+++ /trunk/gdal/port/cpl_vsil_unix_stdio_64.cpp	(revision 38820)
@@ -504,4 +504,5 @@
 #include <linux/fs.h> /* FS_IOC_FIEMAP */
 #ifdef FS_IOC_FIEMAP
+#include <linux/types.h> // for types used in linux/fiemap;h
 #include <linux/fiemap.h> /* struct fiemap */
 #endif
